include/potter/potter.hpp

```diff
@@ -292,6 +292,118 @@ class SharedDataBase {
             }
         }
     }
+
+
+	/*
+	Given the separations and angles, calculate the integrand for B4_1 for a spherically-symmetric potential
+	for an atomic fluid
+	*/
+	void atomic_B4_1_integrand(const double r14, const double r13, const double gama_angle, const double r12, const double eta_angle, double* fval)
+	{
+		// Get the potential function V(r) that we should use
+		auto &pot = this->evaltr.get_potential(0, 0);
+		TEMPTYPE Tstar = this->Tstar; // Local reference just for sharing with the lambda function f
+		auto f = [pot, Tstar](double r) -> TEMPTYPE { return 1.0 - exp(-pot(r) / Tstar); };
+		auto SQUARE = [](double x) { return x * x; };
+		auto sq_r12 = SQUARE(r12);
+		auto sq_r13 = SQUARE(r13);
+		auto sq_r14 = SQUARE(r14);
+		auto rangle_12_13 = sqrt(sq_r12 + sq_r13 - 2 * r12*r13*eta_angle);
+		auto rangle_13_14 = sqrt(sq_r14 + sq_r13 - 2 * r14*r13*gama_angle);
+
+		auto a = sq_r12 * f(r12)*sq_r13*sq_r14*f(r14)*f(rangle_12_13)*f(rangle_13_14);
+
+		if constexpr (std::is_same<decltype(Tstar), double>::value) {
+			// If T is double (real)
+			fval[0] = a;
+		}
+		else if constexpr (std::is_same<decltype(Tstar), std::complex<double>>::value) {
```

Comment:
This is the case for the complex number argument, where you can see that the real part goes into the first index, and the complex part into the second.